### DISCUSSION TITLE
Bring Langfuse to Cloud Marketplaces

### DISCUSSION THREAD

**Discussion by aiakubovich** [langfuse/langfuse#2343] [Ideas]
### Describe the feature or potential improvement

It would be interesting to see Langfuse on cloud marketplaces, particularly on the Microsoft Azure Marketplace. Some competitors already have offerings there. Enterprise support + data not leaving the company cloud, I think, it could be interesting offering.

**Comment by marcklingen** [comment]
Agree, let's track overall interest here

**Comment by marcklingen** [comment]
We have just launched on AWS Marketplace: https://langfuse.com/changelog/2024-09-20-aws-marketplace

**Reply by kespineira** [reply]
@marcklingen has something happened? I don't see that it's available anymore

**Reply by marcklingen** [reply]
hi @kespineira, sorry for the confusion and thank you for flagging this!

We migrated to a new marketplace listing for private offers, it is on our roadmap to offer self-serve marketplace billing. please reach out here if you want to set up marketplace billing for your account: https://langfuse.com/talk-to-us

fixed docs: https://github.com/langfuse/langfuse-docs/pull/3741/changes
